python/cugraph/cugraph/dask/sampling/random_walks.py

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dask.distributed import wait
+
+import dask_cudf
+import cudf
+import operator as op
+
+from pylibcugraph import ResourceHandle
+
+from pylibcugraph.experimental import (
+    uniform_random_walks as pylibcugraph_uniform_random_walks,
+)
+
+from cugraph.dask.comms import comms as Comms
+from cugraph.dask.common.input_utils import get_distributed_data
+
+
+def convert_to_cudf(cp_paths):
+    """
+    Creates cudf Series from cupy arrays from pylibcugraph wrapper
+    """
+
+    print("the cupy paths is \n", cp_paths)
+
+    return cudf.Series(cp_paths)
+
+
+def _call_plc_uniform_random_walks(sID, mg_graph_x, st_x, max_depth):
+
+    return pylibcugraph_uniform_random_walks(
+        resource_handle=ResourceHandle(Comms.get_handle(sID).getHandle()),
+        input_graph=mg_graph_x,
+        start_vertices=st_x,
+        max_length=max_depth,
+    )
+
+
+def random_walks(input_graph, start_vertices, max_depth=None, use_padding=False):
```

Comment:
Wouldn't it be nice to have type annotation for the parameter?

Reply:
Thanks for mentioning this. We will indeed start introducing type annotation in our APIs

Reply:
I filed an [issue](https://github.com/rapidsai/cugraph/issues/2891)